docs/Calendar/BookingsDiagnosticSummary.md

```diff
@@ -0,0 +1,50 @@
+# BookingsDiagnosticSummary
+
+Download the latest release: [BookingsDiagnostic.ps1](https://github.com/microsoft/CSS-Exchange/releases/latest/download/BookingsDiagnosticObjectsSummary.ps1)
```

Comment:
You may want to create an AKA link for this.  Goto aka.ms/aka to create one.

Reply:
aka.ms/BookingsDiagnosticSummary
co-owners shanefe, eric, kenny and denny